.drone.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+set -x
+
+GOOS=linux GOARCH=amd64 CGO_ENABLED=0         go build -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -tags netgo -o release/linux/amd64/drone-hg
+GOOS=linux GOARCH=arm64 CGO_ENABLED=0         go build -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -tags netgo -o release/linux/arm64/drone-hg
+GOOS=linux GOARCH=arm   CGO_ENABLED=0 GOARM=7 go build -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -tags netgo -o release/linux/arm/drone-hg
```

Comment:
we can only build arm and arm64 images for pure Go plugins at this time. This plugin is not pure Go because it uses os.Exec to communicate with mercurial. And mercurial will only run on amd64